dwave/embedding/chain_strength.py

```diff
@@ -37,10 +37,43 @@
 __all__ = ['uniform_torque_compensation', 'scaled']
 
 def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
-    """Chain strength that attempts to compensate for torque that would break
-    the chain.
-
-    The RMS of the problem's quadratic biases is used for calculation.
+    r"""Chain strength that attempts to compensate for chain-breaking torque.
+
+    The problem's connectivity\ [#]_ and quadratic biases are used to calculate
+    a value of chain strength that preforms reasonably well on many problems.
+
+    As the quantum annealing progresses, and the amplitude of the transverse
+    field (:math:`A(s)` in the :ref:`qpu_qa_implementation` section) decreases,
+    the wavefunction representing the QPU's quantum state develops long-range
+    order. A chain strength that increases the correlation of the chains' qubits
+    together with the development of this long-range order produces efficient
+    quantum dynamics [Ray2020]_. Consequently, for hard, frustrated problems
```

Comment:
"Consequently, ..  choose a chain strength that"
I would suggest (because spin-glasses doesn't follow from chain topology):
For many hard frustrated problems (such as spin-glasses), and typical chain topology (path or tree-like chains), an optimal chain strength

Reply:
Updated that, thanks!